google/cloud/composer/README.md

```diff
@@ -55,7 +55,7 @@ int main(int argc, char* argv[]) try {
       composer::EnvironmentsClient(composer::MakeEnvironmentsConnection());
 
   auto const parent =
-      std::string("projects/") + argv[0] + "/locations/" + argv[1];
```

Comment:
ugh, this was me, without a doubt.

Reply:
No worries, that is why you asked for testing.